ClubService.Application/Impl/UpdateTennisClubService.cs

```diff
@@ -92,11 +95,15 @@ public async Task<Guid> UnlockTennisClub(Guid id)
         return id;
     }
     
-    public async Task<Guid> ChangeSubscriptionTier(Guid clubId, Guid subscriptionTierGuid)
+    public async Task<Guid> UpdateTennisClub(Guid id, TennisClubUpdateCommand tennisClubUpdateCommand)
     {
-        var subscriptionTierId = new SubscriptionTierId(subscriptionTierGuid);
-        var tennisClubId = new TennisClubId(clubId);
-        var tennisClub = new TennisClub();
+        if (string.IsNullOrWhiteSpace(tennisClubUpdateCommand.Name) &&
```

Comment:
you meant "String.IsNullOrWhiteSpace" instead of "string.IsNullOrWhiteSpace" right?

Reply:
Rider suggested `string` instead of `String`.